packages/cli/src/commands/new/project.ts

```diff
@@ -107,6 +112,21 @@ export const parseConfig = async (
   flags: Partial<ProjectInitializerConfig>,
   boosterVersion: string
 ): Promise<ProjectInitializerConfig> => {
+  if (flags.default) {
+    return Promise.resolve({
+      projectName: flags.projectName as string,
```

Comment:
Why do we need a cast here?

Reply:
I don't think it's needed, I'll check and remove it if it's not. Thanks!!

Reply:
Oh I saw what happened here. `projectName` is `string | undefined` so it doesn't fit the `ProjectInitializerConfig` if we don't cast it to be a string. There is another way of avoiding this? @alvaroloes

Reply:
🤔  aha! That's because flags type is `Partial<ProjectInitializerConfig>`. Maybe we could create a better type with the name of the project not "partial" but don't worry now about this ;-)

BTW, the exact way to tell Typescript "Believe me, this is not null" is with the bang operator `!`. I think it is cleaner, but I'm not sure if it is discouraged by our linter (although we get the same result)

Reply:
The thing is that this cast would make the CLI try to create a new project named "undefined" if for some reason the project name is not present, so it shouldn't be optional at this point. You can fix it checking that `projectName` is defined before, TypeScript knows that you checked and let you assign the variable without casting. We're checking for its nullity in line 59 of this same file, in the `run` method. You could put the `projectName` in a string typed variable from that point.

Reply:
Hmm I've tried to:
1. Create a constant with the project name and pass it through param (removing it from `ProjectInitializerConfig`). The Script.init(..).step(...).... doesn't accept that.
2. Setting the variable with a cast after the null check. It still says that it should be `string | undefined`

cc @javiertoledo